hs/Server.hs

```diff
@@ -58,6 +63,9 @@ runServer = do
               dir "depts" $ liftIO deptList,
               dir "timesearch" searchResponse,
               dir "calendar" $ lookCookieValue "selected-lectures" >>= calendarResponse,
+              -- dir "draw-json?gid=num" $ look "gid" >>= drawJsonResponse,
```

Comment:
> Should I avoid submitting commented-code in the future?

Master branch code should typically be treated as production code, and as code that other people will have to understand years down the road without your help. When future maintainers look at the commented out code, what could go wrong? What are the benefits to future maintainers? How do future maintainers know these benefits? If you can get access to [Clean Code](http://www.amazon.ca/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882), take a look at the 'commented-out code' section.